### PR TITLE
Update to ruby 3.0.0

### PR DESCRIPTION
**Pull request by naotospace**
Please review.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,40 +1,38 @@
...
   gem 'pry-byebug'
-  gem 'pry-doc', '~> 0.13'
+  gem 'pry-doc'
   gem 'pry-rails'
   gem 'pry-stack_explorer'
   gem 'rails_best_practices'
-  gem 'rspec-rails', '~> 3'
-  gem 'rubocop', '~> 0.58.2', require: false
+  gem 'rspec-rails'
+  gem 'rubocop', require: false
```
Comment (rueyaa332266) [inline Gemfile:35]: Maybe rubocop-rails is also needed?

**Review by rueyaa332266 — CHANGES_REQUESTED** [review]
Please check the comment.

File: Gemfile (hunk 1 above)
Comment (naotospace) [reply Gemfile:35]: Thanks! Fixed.

**Review by rueyaa332266 — APPROVED** [review]
LGTM

**Pull request merged**